azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ChatCompositeView.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.chat.presentation;
+
+import android.content.Context;
+import android.widget.FrameLayout;
+
+import com.azure.android.communication.ui.chat.ChatComposite;
+import com.azure.android.communication.ui.chat.presentation.ui.container.ChatCompositeViewImpl;
+
+/**
+ * Chat composite view.
+ */
+public final class ChatCompositeView extends FrameLayout {
+
+    /**
+     * Creates chat composite view
+     * @param context
+     * @param chatComposite
+     */
+    public ChatCompositeView(final Context context, final ChatComposite chatComposite) {
```

Comment:
will need to sort out constructors. Can be after.

Reply:
don't think we need indirection (ChatCompositeView/ChatCompositeViewImpl).

Reply:
ChatCompositeViewImpl is currently in Kotlin. Decided not to touch it for now, just renamed. Instead create a new one in Java

Reply:
I think that Kotlin is fine for these view classes with standard constructors.

Nobody in Java will ever notice that it's a kotlin class.

Reply:
we can update later though.